cartridges/plugin_page_designer_extend/cartridge/controllers/Home.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+/**
+ * @namespace Home
+ */
+
+const server = require("server");
+server.extend(module.superModule);
+
+const pageMetaData = require("*/cartridge/scripts/middleware/pageMetaData");
+
+/**
+ * Any customization on this endpoint, also requires update for Default-Start endpoint
+ */
+/**
+ * Home-Show : This endpoint is called when a shopper navigates to the home page
+ * @name Base/Home-Show
+ * @function
+ * @memberof Home
+ * @param {category} - non-sensitive
+ * @param {renders} - isml
+ * @param {serverfunction} - append
+ */
+server.append(
```

Comment:
You are modifying the core logic of the controller hence is better to use a replace instead.